content/spire/getting-started.md

```diff
@@ -30,7 +30,7 @@ This guide also illustrates node attestation using join tokens -- a pre-shared k
 
 Finally, this guide assumes the user is running Ubuntu 16.04.
 
-## Step 1 Plan
+## Step 1: Plan {#step-1}
```

Comment:
As you can see here, you can apply custom IDs to headers using `{#...}` syntax. Now you can link to this header using `#step-1`, which is much simpler. Later on, you'll see me use `#step-4` instead of `#step-4-configure-the-server`, for example.

Reply:
LGTM.